data/dashboard.yml

```diff
@@ -90,12 +87,10 @@
     - name: Configuration
       repos:
         - govuk-puppet
-        - fabric-scripts
```

Comment:
Should we remove all the other links from the docs as a part of this PR? There are few manuals that are still linking to the fabric scripts (they'll probably need updating not just removing the links) and styleguides https://github.com/alphagov/govuk-developer-docs/blob/main/source/manual/readmes.html.md?plain=1#L79

Reply:
Definitely - that's a larger piece of work though, captured by https://trello.com/c/AGdifoDb/250-remove-fabric-and-replace-references-in-docs. It's actually looking for a volunteer, if you'd like the job!